io.elementary.code.yml

```
@@ -1,9 +1,7 @@
 app-id: io.elementary.code
-runtime: org.gnome.Sdk # outline plugin needs libvala that is only on sdk
-runtime-version: '3.38'
-base: io.elementary.BaseApp
-base-version: juno-20.08
-sdk: org.gnome.Sdk
+runtime: io.elementary.Platform
```

The need for use the sdk here for outline still
```suggestion
runtime: io.elementary.Sdk
```